inc/define.php

```diff
@@ -510,3 +511,16 @@
 //Maximum time, in miliseconds a saved search should not exeed
 //so we count it on display (using automatic mode).
 $CFG_GLPI['max_time_for_count'] = 200;
+
+$CFG_GLPI["impact_asset_types"]  = [
+   'Computer'         => "pics/impact/computer.png",
```

Comment:
Is this really required to list all images? Using lowered type name seems enough

Reply:
At first we used lowecase itemtype name but we switched to this as it makes it easier for plugins to add their own types and images (or to modify the existing ones).

Reply:
or to modify the existing ones => I'm pretty sure this is a bad idea